### ISSUE TITLE
Missing file in Wikipedia raw text

### ISSUE THREAD

**Issue by mickvanhulst** [dalab/deep-ed#26]
Dear author,

As I'm running the file to generate all datasets, I run into an error when running gen_test_train/gen_all.lua. Here the part regarding Wikipedia goes wrong as it is missing a file. I am missing the following file (which is not in the provided dataset):
./data/basic_data/test_datasets/wned-datasets/wikipedia/RawText/Chippenham_United_F.C.

I tried taking the approach that the issue listed [here](https://github.com/dalab/deep-ed/issues/2) had taken, but in that case it was missing a different file:
./data/basic_data/test_datasets/wned-datasets/wikipedia/RawText/Electoral_division_of_Apsley'

Did you skip over documents that did not exist or could you please tell me how you handled this?

Thanks in advance!

**Comment by octavian-ganea** [comment]
Hmm, I am not sure why this happens. As far as i remember (might be wrong though), I skipped only over docs that have no annotations.

**Comment by mickvanhulst** [comment]
I found a similar [issue ](https://github.com/dalab/deep-ed/issues/15). I had to perform two operations:
1) unzip it using my Ubuntu instance (I use Windows by default).
2) Windows recognizes dots at the end of filenames as file extensions, so I changed the files by removing the dot at the end (do not forget to change the XML file as well). I think this is purely a Windows problem, thanks for your quick response!
